linera-views/src/journaling.rs

```diff
@@ -200,7 +200,26 @@ where
     K: DirectKeyValueStore + Send + Sync,
     K::Error: From<JournalConsistencyError>,
 {
-    /// Resolves the database by using the header that has been retrieved
+    /// Resolves the pending operations that were previously stored in the database
+    /// journal.
+    ///
+    /// For each block processed, we atomically update the journal header as well. When
+    /// the last block is processed, this atomically clears the journal and make the store
+    /// finally available again (for the range of keys managed by the journal).
+    ///
+    /// This function respects the constraints of the underlying key-value store `K` if
+    /// the following conditions are met:
+    ///
+    /// (1) each block contains at most `K::MAX_BATCH_SIZE - 2` operations;
+    ///
+    /// (2) the total size of the all operations in a block doesn't exceed:
+    ///     `K::MAX_BATCH_TOTAL_SIZE - sizeof(block_key) - sizeof(header_key) - sizeof(bcs_header)`
+    ///
+    /// (3) every operation in a block satisfies the contraints on individual database operations
+    ///     represented by `K::MAX_KEY_SIZE` and `K::MAX_VALUE_SIZE`.
+    ///
+    /// (4) `block_key` and `header_key` don't exceed `K::MAX_KEY_SIZE` and `bcs_header`
+    /// doesn't exceed `K::MAX_VALUE_SIZE`.
```

Comment:
```suggestion
    ///     doesn't exceed `K::MAX_VALUE_SIZE`.
```

(Or the other way round. Indentation is also a bit inconsistent in the other doc comments.)